### ISSUE TITLE
Near Future Launch Vehicles Tanks / Tweakscale RealFuels 

### ISSUE THREAD

**Issue by mikken40** [TweakScale/Companion_FuelSwitches#7]
Hello,

I use Realism Overhaul and RealFuels.
I have created a patch for the NFLV tanks so that I can also use RealFuels with them. That works great so far.
But the problem is that I can no longer scale these tanks with Tweakscale?
What could be the reason for this? I have already read that RealFuels in conjunction with Tweakscale causes problems for some. And it has been deactivated in the RealFuels settings.
However, I would like to test this myself and cannot find the setting to reactivate Tweakscale for RealFuels.

I thank you in advance for any comment and hope it is described clearly ;)

LG Mike

**Comment by Lisias** [comment]
Transferring the issue to the proper place.

TweakScale (the main distribution) only supports Stock and DLCs, anything else is handled by a Companion.

The Companion that will handle Real Fuels is this one.

**Comment by Lisias** [comment]
@mikken40 , exactly what do you mean to "been deactivated"?

The part lost the scaling capability, or it is not being scaled correctly?

**Comment by Lisias** [comment]
Now I understand. 0n https://github.com/TweakScale/TweakScale/discussions/326 the user explained that it can't scale the part after adding support for Real Fuels.

TL;DR: Real Fuel support is in need to be implemented.

The Companions were kinda stalled since I got screwed by https://github.com/TweakScale/TweakScale/issues/307 - there's no point into adding support for 3rd parties when I was unable to correctly understand scaling Stock.

Since https://github.com/TweakScale/TweakScale/issues/307 is now under control (the `PartModule` is being instantiated with `enable = true` when it used to be `false` since 1.9.0), it's now plausible to go back to implementing the Companions.

I will check this issue this WeekEnd - RealFuels, **in theory** should be working.
